packer/windows.pkr.hcl

```diff
@@ -134,12 +129,6 @@ build {
       "[Environment]::SetEnvironmentVariable('PATH',  \"C:\\strawberry\\${local.perl_version}\\perl\\bin;\" + [Environment]::GetEnvironmentVariable('PATH', 'Machine'), 'Machine')",
     ]
   }
-
-  # install openssl
-  provisioner "powershell" {
```

Comment:
I think for an easier transition of CI, it'd be better if we left the current environment alone. Otherwise we'll need to modify the CI image and the CI tasks in the postgres repo synchronously. And different repos will all be broken.